src/wasm/wasm-debug.cpp

```diff
@@ -473,12 +473,17 @@ struct LocationUpdater {
   AddrExprMap oldExprAddrMap;
   FuncAddrMap oldFuncAddrMap;
 
+  // Map offsets of location list entries in the debug_loc section to the index
+  // of their compile unit.
+  std::map<BinaryLocation, size_t> locToUnitMap;
+
   // Map start of line tables in the debug_line section to their new locations.
   std::map<BinaryLocation, BinaryLocation> debugLineMap;
 
-  // Map offset of location list entries in the debug_loc section to the base
-  // address of the compilation units referencing them.
-  std::map<BinaryLocation, BinaryLocation> debugLocToUnitMap;
+  typedef std::pair<BinaryLocation, BinaryLocation> OldToNew;
+
+  // Map of compile unit index => old or new base offsets.
```

Comment:
by "old or new" you mean that it starts out with the old data and we update it?

Reply:
Oh, the `or` should be `and`. Fixing and elaborating.